### PR TITLE
fix json and add clarity about what's required

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: discovery/spec.md [hunk 1]
```diff
@@ -588,10 +588,17 @@ The result MUST be a JSON object of the following form:
 ```
 {
   "servicefilterattributes": [ "name", ... ],
-  "pagination": "true"
+  "pagination": true
 }
 ```
 
+The following additional constraints apply:
+- The list of feature names are case sensitive.
```
Comment (jskeet) [inline discovery/spec.md:596]: are => is? Or rewrite.

File: discovery/spec.md [hunk 2]
```diff
@@ -588,10 +588,17 @@ The result MUST be a JSON object of the following form:
...
+- The `servicefilterattribute` MUST be present, and MUST have at least the
```
Comment (jskeet) [inline discovery/spec.md:597]:
```suggestion
- The `servicefilterattributes` property MUST be present, and MUST have at least the
```

File: discovery/spec.md [hunk 3]
```diff
@@ -588,10 +588,17 @@ The result MUST be a JSON object of the following form:
...
+  `name` attribute in its list. The list of attribute names are case sensitive.
```
Comment (jskeet) [inline discovery/spec.md:598]: Remove the second sentence here, given that it's already above?

File: discovery/spec.md (hunk 1 above)
Comment (duglin) [reply discovery/spec.md:596]: "fearures" -> "features" on 585

Comment (duglin) [reply discovery/spec.md:596]: went with "the feature names are case sensitive"

File: discovery/spec.md (hunk 2 above)
Comment (duglin) [reply discovery/spec.md:597]: fix

Comment (duglin) [reply discovery/spec.md:597]: I think this is different. This is talking about filter attribute names, but the previous one is talking about feature names. But I did reword the sentence per your previous comment.

File: discovery/spec.md (hunk 3 above)
Comment (duglin) [reply discovery/spec.md:598]: I think this is different. This is talking about filter attribute names, but the previous one is talking about feature names. But I did reword the sentence per your previous comment.

**Comment by duglin** [comment]
Approved on the 12/02 call w/Jon's minor edits

**Pull request merged**